markdown/en/ff-checkout-tracking.md

```diff
@@ -0,0 +1,71 @@
+## Checkout Tracking
```

Comment:
This looks very similarly to the checkout tracking part in tracking guide. Maybe we should just put a link to this file there instead of maintaining duplicated docs?